src/traversals/parallel_bfs.jl

```diff
@@ -15,7 +15,7 @@ import Base: push!, shift!, isempty, getindex
 
 export bfs_tree, LevelSynchronousBFS
 
-mutable struct LevelSynchronousBFS <: AbstractGraphVisitAlgorithm end
+struct LevelSynchronousBFS end
```

Comment:
It has to be threaded because it uses atomics. You can't synchronize multiple processes with atomic instructions.